### PR TITLE
i18n(hi): Add Hindi language

### PR DESCRIPTION
**Pull request by at-the-vr**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- `hi.json` added for Hindi language option
- `translations/index.ts` updated to include hindi object/json

### CODE REVIEW

File: packages/starlight/translations/hi.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "skipLink.label": "इसे छोड़कर कंटेंट पर जाएं",
+  "search.label": "खोजें",
+  "search.shortcutLabel": "(खोजने के लिए / दबाएँ)",
+  "search.cancelLabel": "रद्द करे",
+  "search.devWarning": "खोज केवल उत्पादन बिल्ड में उपलब्ध है। \nस्थानीय स्तर पर परीक्षण करने के लिए साइट बनाए और उसका पूर्वावलोकन करने का प्रयास करें।",
+  "themeSelect.accessibleLabel": "थीम चुनें",
+  "themeSelect.dark": "अँधेरा",
+  "themeSelect.light": "रोशनी",
+  "themeSelect.auto": "स्वत:",
```
Comment (delucis) [inline packages/starlight/translations/hi.json:10]: Should the `:` be here?

```suggestion
  "themeSelect.auto": "स्वत",
```

File: packages/starlight/translations/hi.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "languageSelect.accessibleLabel": "भाषा चुने",
+  "menuButton.accessibleLabel": "मेन्यू",
+  "sidebarNav.accessibleLabel": "मुख्य",
+  "tableOfContents.onThisPage": "इस पृष्ठ पर",
+  "tableOfContents.overview": "अवलोकन",
+  "i18n.untranslatedContent": "यह कंटेंट अभी तक आपकी भाषा में उपलब्ध नहीं है.",
+  "page.editLink": "पृष्ठ संपादित करें",
+  "page.lastUpdated": "आखरी अद्यतन:",
```
Comment (delucis) [inline packages/starlight/translations/hi.json:18]: Feel free to ignore this — I have no way of knowing if it’s correct — but when checking these with Google Translate, it wanted to change this:
```suggestion
  "page.lastUpdated": "आखिरी अद्यतन:",
```

**Review by delucis — COMMENTED** [review]
Thank you @at-the-vr! Happy to have you in the Starlight repo ☺️ 

Left a couple of comments.

**Comment by at-the-vr** [comment]
Thanks for the suggestions. 💯 valid

**Pull request merged**